From: Beren Minor <beren.minor+git@gmail.com>
Date: Sun, 3 Jun 2012 15:27:06 +0200
Subject: DEBIAN: Set the SDL_VIDEO_X11_WMCLASS environment variable with the
 executable name, so gnome shell and others recognize the
 application and show the right icon.

---
 gemrb/GemRB.cpp |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/gemrb/GemRB.cpp b/gemrb/GemRB.cpp
index 6079424..36c65c9 100644
--- a/gemrb/GemRB.cpp
+++ b/gemrb/GemRB.cpp
@@ -64,6 +64,8 @@ static void appPutToForeground()
 
 int main(int argc, char* argv[])
 {
+	setenv("SDL_VIDEO_X11_WMCLASS", argv[0], 0);
+
 #ifdef M_TRIM_THRESHOLD
 // Prevent fragmentation of the heap by malloc (glibc).
 //
